data/src/main/java/ca/bc/gov/data/model/mapper/ResponseToDtoMapper.kt

```diff
@@ -86,3 +92,15 @@ fun DispensingPharmacy.toDispensingPharmacyDto(medicationRecordId: Long) = Dispe
     phoneNumber = phoneNumber,
     faxNumber = faxNumber
 )
+
+fun Media.toMediaMetaData(): MediaMetaData = MediaMetaData(
+    mediaType = mediaType ?: throw MyHealthException(SERVER_ERROR, "Invalid Response"),
```

Comment:
MyHealthNetworkException suits best here.

Reply:
Intentionally I have chosen this exception & will also do gradual refactoring this exception handling bit.